action.js

```diff
@@ -31,7 +42,8 @@ async function run(octokit, context) {
 	const plugin = new SizePlugin({
 		compression: getInput('compression'),
 		pattern: getInput('pattern') || '**/dist/**/*.js',
-		exclude: getInput('exclude') || '{**/*.map,**/node_modules/**}'
+		exclude: getInput('exclude') || '{**/*.map,**/node_modules/**}',
+		stripHash: stripHash(getInput('stripHash'))
```

Comment:
to be consistent with e.g. `getInput('build-script')`, should we use a dasherized option name?

```suggestion
		stripHash: stripHash(getInput('strip-hash'))
```